src/Mapper/Arrayizer/AddressArrayizer.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * This file is part of byrokrat\giroapp.
+ *
+ * byrokrat\giroapp is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * byrokrat\giroapp is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with byrokrat\giroapp. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright 2016-17 Hannes Forsgård
+ */
+
+declare(strict_types = 1);
+
+namespace byrokrat\giroapp\Mapper\Arrayizer;
+
+use byrokrat\giroapp\Model\PostalAddress;
+
+/**
+ * Takes a Address object and transforms it to an array
+ */
+
+class AddressArrayizer
+{
+    /**
+     * @var PostalAddress
+     */
+    private $address;
+
+    public function __construct(PostalAddress $address)
```

Comment:
Let `$address` be an argument to `getArray()` (or I would call the method `toArray()` in this situation, to clearify that it takes something and transforms it). That way the arrayier is reusable.